### PR TITLE
Add VibeVoice-1.5B perf benchmark and wire to nightly

### PR DESCRIPTION
**Pull request by saiarthiraguram**
### Ticket
[Link to Github Issue](https://github.com/tenstorrent/tt-xla/issues/5217)

### Problem description

VibeVoice-1.5B (`microsoft/VibeVoice-1.5B`) was brought up single-device on n150, but it has no perf benchmark and is not part of the nightly performance pipeline, so its throughput/PCC is not tracked over time.

### What's changed

Adds a perf benchmark for VibeVoice-1.5B and wires it into the nightly perf matrix.

- **`tests/benchmark/test_encoders.py::test_vibevoice`** — runs the model through the generic single-forward + PCC encoder harness. VibeVoice's bringup forward reduces to the Qwen2.5 LM backbone producing logits (`speech_tensors=None`; the semantic connector is exercised but unused), and the loader wraps the model to return a bare logits tensor, so it fits the existing harness without changes. Config: bf16, batch 1, seq len 32, loop count 32, optimization level 1, trace disabled.
- **`.github/workflows/perf-bench-matrix.json`** — adds a `vibevoice` entry pinned to `runs-on: n150-perf` (the verified bringup arch). This matrix is filtered and executed by the nightly pipeline (`schedule-nightly.yml` → `perf-benchmark` → `call-filtered-perf-tests.yml`).

**Impact:** VibeVoice-1.5B throughput and PCC are now tracked in the nightly benchmark report on n150.

**Dependency:** the benchmark imports the VibeVoice loader from `third_party.tt_forge_models.vibevoice`. It only runs green once the tt-forge-models loader PR (branch `sai_arthi_raguram/vibe_voice`) lands and the submodule is uplifted in tt-xla. Land + uplift that first.

### Checklist
- [x] New/Existing tests provide coverage for changes — `tests/benchmark/test_encoders.py::test_vibevoice` verified on n150: **PCC=0.992910**, `1 passed in 197s`.

## Logs
[benchmark_vibevoice_n150.log](https://github.com/user-attachments/files/29243310/benchmark_vibevoice_n150.log)
[bringup_steps.txt](https://github.com/user-attachments/files/29243312/bringup_steps.txt)

### CODE REVIEW

**Review by vvukomanTT — APPROVED** [review]

**Review by nsumrakTT — APPROVED** [review]

File: .github/workflows/perf-bench-matrix.json [hunk 1]
```diff
@@ -119,6 +119,11 @@
         "pyreq": "transformers==4.57.1 FlagEmbedding",
         "pytest": "tests/benchmark/test_encoders.py::test_bge_m3"
       },
+      {
+        "name": "vibevoice",
+        "pytest": "tests/benchmark/test_encoders.py::test_vibevoice",
+        "runs-on": "n150-perf"
+      },
```
Comment (vkovacevicTT) [inline .github/workflows/perf-bench-matrix.json]: Please rebase and add `"group" : "experimental"` here :)

**Review by vkovacevicTT — CHANGES_REQUESTED** [review]

File: .github/workflows/perf-bench-matrix.json (hunk 1 above)
Comment (saiarthiraguram) [reply .github/workflows/perf-bench-matrix.json]: Added the requested change @vkovacevicTT

**Comment by saiarthiraguram** [comment]
close as superseded by [#6042](https://github.com/tenstorrent/tt-xla/pull/6042).
